t/api/11-bugs.t

```diff
@@ -77,4 +83,15 @@ $t->post_ok('/api/v1/jobs/99926/comments', form => {text => 'wicked bug: jsc#SLE
 $get = $t->get_ok('/api/v1/bugs/3');
 is($get->tx->res->json->{bugid}, 'jsc#SLE-42999', 'Bug was created by comment post');
 
+$t->post_ok('/api/v1/bugs', form => {title => "new", bugid => 'bsc#123'});
+my $bugid = $t->tx->res->json->{id};
+$t->app->schema->resultset('Bugs')->find($bugid)->update(
+    {
+        t_created => time2str('%Y-%m-%d %H:%M:%S', time - 500, 'UTC'),
+    });
+$t->get_ok('/api/v1/bugs?created_since=500');
+is(keys %{$t->tx->res->json->{bugs}}, 3, 'All reported bugs');
```

Comment:
Don't call `keys` in list context here, it only works because `is()` has a magical prototype that forces the first argument into scalar context. That's not something you should rely on.